qi-test/tests/compiler/rules/private/deforest-util.rkt

```diff
@@ -20,4 +20,4 @@
   (string-contains? (format "~a" exp) "filter-cstream"))
 
 (define (car-deforested? exp)
-  (string-contains? (format "~a" exp) "car-cstream"))
+  (string-contains? (format "~a" exp) "cad*r-cstream"))
```

Comment:
Assuming this fixes the rolled-up version of 488fd48 (Deforest cadr, caddr,
cadddr, caddddr, cadddddr., 2024-03-11) (including typo fixes, new names, etc.),
it should also be part of that commit.